admin_sdk/directory/quickstart.py

```diff
@@ -14,33 +14,45 @@
 
 # [START admin_sdk_directory_quickstart]
 """
-Shows basic usage of the Admin SDK Directory API. Lists of first 10 users in the
-domain.
+Shows basic usage of the Admin SDK Directory API. Lists of first 10 users in
+the domain.
 """
 from __future__ import print_function
 from apiclient.discovery import build
 from httplib2 import Http
 from oauth2client import file, client, tools
 
-# Setup the Admin SDK Directory API
+# If modifying these scopes, delete the file token.json.
 SCOPES = 'https://www.googleapis.com/auth/admin.directory.user'
-store = file.Storage('token.json')
-creds = store.get()
-if not creds or creds.invalid:
-    flow = client.flow_from_clientsecrets('credentials.json', SCOPES)
-    creds = tools.run_flow(flow, store)
-service = build('admin', 'directory_v1', http=creds.authorize(Http()))
-
-# Call the Admin SDK Directory API
-print('Getting the first 10 users in the domain')
-results = service.users().list(customer='my_customer', maxResults=10,
-                               orderBy='email').execute()
-users = results.get('users', [])
-
-if not users:
-    print('No users in the domain.')
-else:
-    print('Users:')
-    for user in users:
-        print('{0} ({1})'.format(user['primaryEmail'], user['name']['fullName']))
+
+
+def main():
+    """Shows basic usage of the Google Admin SDK Directory API.
+
```

Comment:
Remove extra \n.

Reply:
This is required by Google Python style: https://github.com/google/styleguide/blob/gh-pages/pyguide.md#35-blank-lines